forest/src/daemon.rs

```diff
@@ -123,6 +127,9 @@ pub(super) async fn start(config: Config) {
                     sync_state,
                     network_send,
                     network_name,
+                    heaviest_tipset,
+                    subscriber,
```

Comment:
This subscriber is cloned here, but never polled off of, so this will stall eventually (need to be certain no subscriber _doesn't_ get polled

Reply:
What do you mean? It's being polled in the sub_head_changes. Maybe im missing something?

Reply:
but that's a cloned subscriber?